payments-model/src/main/java/com/stripe/android/model/ConsumerPaymentDetails.kt

```diff
@@ -73,6 +73,7 @@ data class ConsumerPaymentDetails(
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     data class Passthrough(
         override val id: String,
+        val paymentMethodId: String,
```

Comment:
👀 This `id` was being set to the payment method ID, when it _should_ be the payment details ID. Things (mostly) only worked because we were only using the `id` as the payment method ID when the object type as `Passthrough`.

The fix is to correctly set the `id` to the payment details ID and have a separate `paymentMethodId`.